src/EditorFeatures/Core/Implementation/BraceMatching/ClassificationTypeDefinitions.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Utilities;
 
+#pragma warning disable 649
```

Comment:
Actually, why did this change introduce a new warning?  Can't the compiler see the generated IVT attributes?